projects/ion/src/lib/triple-toggle/triple-toggle.component.scss

```
@@ -0,0 +1,58 @@
+@import '../../styles/index.scss';
+
+.ion-triple-toggle {
+  display: flex;
+  align-items: center;
+
+  ion-button:nth-child(1) ::ng-deep {
+    button {
+      border-top-right-radius: 0 !important;
+      border-bottom-right-radius: 0 !important;
+    }
+  }
+
+  ion-button:nth-child(2) ::ng-deep {
+    button {
+      border-radius: 0 !important;
+      margin-left: -1px;
+      margin-right: -1px;
+      position: relative;
+      z-index: 1;
+    }
+  }
+
+  ion-button:nth-child(3) ::ng-deep {
+    button {
+      border-top-left-radius: 0 !important;
+      border-bottom-left-radius: 0 !important;
+    }
+  }
+
+  ion-button ::ng-deep {
+    .ion-btn-primary {
+      border: 1px solid $primary-6;
+
+      &:hover {
+        border-color: $primary-5;
+      }
+
+      &:active {
+        border-color: $primary-7;
+      }
+
+      &:disabled {
+        border-color: $neutral-4;
+      }
```

```suggestion
      border: 1px solid transparent;
```
If you add a transparent border, you wont have to set all the other states again.

---

Interessante!